full_import.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
 
-until host ${1} &> /dev/null
+retryCount=0
+
+# run the script as sudo so that you
+if [ -z "$SUDO_COMMAND" ]
+then
+  echo -e "Only root can run this script.\nRelaunching script with sudo.\n"
+  sudo -E $0 $*
+  exit 0
+fi
+
+until [ host ${1} &> /dev/null -a $retryCount -lt 5 ]
 do
   echo "Waiting for environment..."
   sleep 20
+  sudo pkill -HUP -x mDNSResponder
```

Comment:
Clear the local dns cache in case the host lookup fail is cached from the last run